icon/data.h

```diff
@@ -1,247 +1,17 @@
 #ifndef ICON_DATA_H
 #define ICON_DATA_H
 
+#include "../nsxiv.h"
```

Comment:
Why is this needed?

Reply:
Compiles fine without, so removed it.

Reply:
> Why is this needed?

That was just to silence some LSP errors, but yeah, it compiles fine without it.